public/app/screens/main/sections/settings/user-info/user-info.jsx

```diff
@@ -0,0 +1,71 @@
+var React = require('react');
+var Reflux = require('reflux');
+var styleMixin = require('mixins/style-mixin');
+
+var userDataStore = require('stores/user-data');
+
+var UserInfo = React.createClass({
+
+  mixins: [
+    styleMixin(require('./style.scss')),
+    Reflux.listenTo(userDataStore, 'onUserDataChanged')
+  ],
+
+  getInitialState: function getInitialState () {
+    return {
+      model: {},
+      error: {}
+    };
+  },
+
+  onUserDataChanged: function onUserDataChanged (userData) {
+    if (userData.model) {
+
+      this.setState({
+        model: userData.model
+      });
+    } else if (userData.error) {
+      // TODO we can add more error handling login for the component
+      this.setState({
+        error: userData.error
+      });
+    }
+  },
```

Comment:
You are not consistent with your method order. Sometimes getInitialState is the first, sometimes an event handler